lib/evss/disability_compensation_form/service.rb

```diff
@@ -33,13 +33,13 @@ def submit_form(form_content)
       private
 
       def headers_for_user(user)
-        EVSS::DisabilityCompensationAuthHeaders.add_headers(EVSS::AuthHeaders.new(user).to_h, user)
+        EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
       end
 
       def handle_error(error)
         if error.is_a?(Common::Client::Errors::ClientError) && error.status != 403 && error.body.is_a?(Hash)
-          log_message_to_sentry(
```

Comment:
this log_message_to_sentry can stay it's just the auth headers one that was having an issue